emrun.py

```diff
@@ -1385,7 +1386,7 @@ def get_system_info(format_json):
       return info.strip()
   else:
     try:
-      unique_system_id = open(os.path.expanduser('~/.emrun.generated.guid'), 'r').read().strip()
```

Comment:
This file is designed to be independent of shared.py.   So you use the long form for defined a duplicate copy of the helper here.

Reply:
actually this file is also designed to work with python2 apparently (although we don't have any tests for that.. we should add one at some point).

Reply:
OK, thanks. I went for the open context manager, which is still simpler that re-defining a helper function for 1 occurrence.